### ISSUE TITLE
Android crashing with exception 'get for prop 'AppboyReactBridge': java.lang.NoClassDefFoundError'

### ISSUE THREAD

**Issue by sweetkat** [Appboy/appboy-react-sdk#138]
Android react native app started crashing with an error `Error: Exception in HostObject::get for prop 'AppboyReactBridge': java.lang.NoClassDefFoundError: Failed resolution of: Lcom/appboy/models/outgoing/AppboyProperties;, js engine: hermes` on latest `react-native-appboy-sdk@1.33.1`.

Started going through the Android SDK change list and came across the changes for SDK 14
https://github.com/Appboy/appboy-android-sdk/blob/master/CHANGELOG.md#1400

The only way I could get around the crashing android is to patch `react-native-appboy-sdk` in node modules with an updated dependency.

```
com.appboy.models.outgoing.AppboyProperties -> com.braze.models.outgoing.BrazeProperties
```
https://github.com/Appboy/appboy-react-sdk/blob/master/android/src/main/java/com/appboy/reactbridge/AppboyReactBridge.java#L21

Would really appreciate some comments on how to better address the issue or I can create a PR to update the dependency.

**Comment by radixdev** [comment]
Hi @sweetkat ,

We will be fixing this issue very shortly. Thanks for filing!

**Comment by mikefausz** [comment]
@radixdev I'm seeing the same error on `react-native-appboy-sdk@1.32.0`. Any idea when we can expect a patch for this? And how we should fix it in the meantime?

**Comment by wesleyorbin** [comment]
@mikefausz  @sweetkat We just released v1.34.0 which contains a fix for this release. I'm going to close out the issue, but feel free to re-open if you're having any issues.
